examples/flows/chat/chat-math-variant/README.md

```diff
@@ -0,0 +1,50 @@
+# Test your prompt variants for chat with math
+This is a prompt tuning case with 3 prompt variants for math question answering.
```

Comment:
Can you elaborate more in somewhere to let users know how to do prompt tunning for this case? Otherwise, I don't see its big different compare with basic-char or chat-with-wikipedia,  it's still a standard introduction with another chat flow. In my understanding this case plus the evaluation flow is to show how to tune a chat flow

Reply:
This example aims to emphasize the importance of batch testing and evaluation in PromptFlow, using the most basic chat scenario as a foundation. 

Our aim is to enable users to quickly grasp the advantages of prompt tuning and experimentation with pf.

We provided [tutorial.md](https://github.com/microsoft/promptflow/pull/524/files?short_path=4071601#diff-40716014562432ce15ecf45dcc28e5b1909317cc631e7d2174e29ffc51252127),  introducing the prompt engineering and tunning approcah (Chain of thought). Here, we offer an in-depth explanation and introduction to the case. Moreover, we have produced a video demonstration that showcases the development of the flow using the VSC extension.

Reply:
Thanks for the explain, I mean if user see the example directly , maybe they don't know the benefit of it. What about add a refer link?

Reply:
Good point, it would be better to add a link to the tutorial to the process readme @ChenJieting let's do it

Reply:
I have added more explanation in chat-math-variant/README.md file.